source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,79 @@
+.. _ruby-quick-start-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: installation, setup, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install dependencies
+
+      Before you begin developing, ensure you have the following dependencies
+      installed in your development environment:
+      
+      - `Ruby <https://www.ruby-lang.org/en/downloads/>`__ version 2.7
+        or later
+      - `bson <https://rubygems.org/gems/bson>`__ version 4.14.1 or later, up to
+        version 6.0
+
+      .. note::
+
+         {+language+} is pre-installed on macOS and some Linux distributions,
+         but the system version might be outdated. You can use a package 
+         manager, such as `Homebrew <https://formulae.brew.sh/formula/ruby>`__,
+         to install the latest {+language+} version.
+
+   .. step:: Install the {+driver-short+}
+
+      From your shell, run the following command to install
+      the {+driver-short+}:
+
+      .. code-block:: bash
+         
+         gem install mongo
+
+      This command installs version {+version-number+} of
+      the driver.
+
+   .. step:: Create a project directory
+
+      Run the following command in your shell to create a directory
+      called ``ruby-quickstart`` for this project:
+
+      .. code-block:: bash
+
+         mkdir ruby-quickstart
+
+      Select the tab corresponding to your operating system and run the following commands
+      to create a ``quickstart.rb`` file in the ``ruby-quickstart`` directory:
+
+      .. tabs::
+
+          .. tab:: macOS / Linux
+             :tabid: create-file-mac-linux
+
+             .. code-block:: bash
+
+                cd ruby-quickstart
+                touch quickstart.rb
+
+          .. tab:: Windows
```

Comment:
We do not officially support Ruby driver on Windows, so I am not sure if Windows should be mentioned in the getting started.

Reply:
Added a note about this